studio/stores/pgmeta/MetaStore.ts

```diff
@@ -60,6 +60,8 @@ export interface IMetaStore {
   publications: IPublicationStore
   types: IPostgresMetaInterface<any>
 
+  projectRef?: string
```

Comment:
This is a precursor to getting rid of the `reaction` in `RootStore`. I didn't go all the way here because that is actually more of a refactor than it seems! More discussion is needed on a new app-wide loading strategy before we can make the full change